tests/library/capabilities.spec.ts

```diff
@@ -97,9 +95,9 @@ it('should play webm video @smoke', async ({ page, asset, browserName, platform,
 });
 
 it('should play audio @smoke', async ({ page, server, browserName, platform }) => {
-  it.fixme(browserName === 'firefox' && platform === 'win32', 'https://github.com/microsoft/playwright/issues/10887');
-  it.fixme(browserName === 'firefox' && platform === 'linux', 'https://github.com/microsoft/playwright/issues/10887');
-  it.fixme(browserName === 'webkit' && platform === 'win32', 'https://github.com/microsoft/playwright/issues/10892');
+  it.fixme(browserName === 'webkit' && platform !== 'darwin');
```

Comment:
it wasn't fixme on linux before?

lets try to stick with === rather than !== I think thats easier to read.